openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -358,6 +359,13 @@ def schedules_for_bin(self):
                 )
                 # continue to the next schedule, don't yield an email for this one
             else:
+                unsubscribe_url = None
+                if 'bulk_email_optout' in settings.ACE_ENABLED_POLICIES:
```

Comment:
I would suggest using a waffle flag rather than checking in settings.

Reply:
@awaisdar001 I would prefer a waffle flag too, but since [`bulk_email_optout` is already a default part of the `settings.ACE_ENABLED_POLICIES`](https://github.com/edx/edx-platform/blob/a439d5164c07e4695181b15244e8e5c7681421c4/openedx/core/djangoapps/ace_common/settings/common.py#L5-L7), we thought we should use what's there.

Is that ok?

Reply:
Thank you for providing information that is already a default part of settings. What that means is, the change would reflect on other platforms using weekly highlight? Correct? 
With the waffle flag, it would be easier to switch on / off in different platforms. 

Let me know what you think.

Reply:
bc01dc5e290e992b986aeaaddf73ff150f80cc98 added a waffle flag to enable the unsubscribe link in the scheduled emails.